.github/workflows/screenshot.yml

```diff
@@ -0,0 +1,65 @@
+name: Screenshot
+
+on: [push]
+
+jobs:
+  screenshot:
+    runs-on: ubuntu-latest
+    env:
+      BASEMAPS_BUCKET: linz-basemaps
+
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: linz/action-typescript@v1
+
+      # Package all the files
+      - name: Bundle & Package all files
+        run: |
+          npx lerna run bundle --stream
+          npm pack --workspaces
+        env:
+          NODE_ENV: "production"
+
+      - name: (NonProd) Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1.6.1
+        with:
+          aws-region: ap-southeast-2
+          mask-aws-account-id: true
+          role-to-assume: ${{ secrets.AWS_ROLE_NON_PROD }}
+      
+      - name: (Screenshot) Screenshot Pull Request Changes
+        run: |
+          ./packages/cli/bin/bmc.js screenshot --config s3://${BASEMAPS_BUCKET}/config-latest.json.gz --assets s3://${BASEMAPS_BUCKET}/assets-latest.tar.co --tiles packages/cli/test-tiles/default.test.tiles.json --output .artifacts/visual-snapshots
```

Comment:
@blacha This need the latest config. I think we can allow screenshot allow default to get the `cb_latest` if no config passed in?